src/cdk/private/private.md

```diff
@@ -0,0 +1 @@
+Private utilities meant to be used only within the CDK and Material.
```

Comment:
This makes me wonder if this sort of utility is useful to other Angular component libraries. Maybe over time we could graduate these things to the framework or CDK